conf/config.yml

```diff
@@ -69,6 +58,26 @@ http:
 #
 owner_id_var: FLOTILLA_RUN_OWNER_ID
 
+ecs:
+  log:
+  namespace: flotilla-os-logs
+  retention_days: 90
+  driver:
+    name: awslogs
+    options:
+      awslogs-group: flotilla-os-logs
+      awslogs-stream-prefix: flotilla
+      awslogs-region: us-east-1
+
+eks:
+  log:
+    namespace: flotilla-eks-namespace
```

Comment:
What is the purpose of the log-group?

Reply:
Fallback if namespace is not set (per the old ECS cloudwatch client logs)